requirements.txt

```diff
@@ -4,52 +4,53 @@
 #
 #    pip-compile --output-file=requirements.txt requirements.in setup.py
 #
-amqp==5.1.0               # via kombu
+amqp==5.1.1               # via kombu
 appdirs==1.4.4            # via fs, snakemake
 attrs==21.4.0             # via jsonschema
 bravado-core==5.17.0      # via bravado
 bravado==10.3.2           # via reana-commons
 certifi==2021.10.8        # via requests
 charset-normalizer==2.0.12  # via requests
 checksumdir==1.1.9        # via reana-commons
-click==8.1.1              # via reana-commons
+click==8.1.3              # via reana-commons
 configargparse==1.5.3     # via snakemake
-connection-pool==0.0.3    # via snakemake
```

Comment:
Interesting this `foo-bar` vs `foo_bar` dance... Is this due to Linux vs macOS?  Are you using `pip-tools==5.4.0` and `python3.8` locally when compiling the list of requirements?

Reply:
For me it's the same as in this PR using Linux, `pip-tools==5.4.0` and `python3.8`

Reply:
It happens with macOS and when using the `python:3.8` docker image, in both cases using `pip-tools==5.4.0`